static/templates/settings/user_list_admin.hbs

```diff
@@ -1,7 +1,12 @@
 <div id="admin-user-list" class="settings-section" data-name="user-list-admin">
     <h3 class="inline-block">{{t "Users" }}</h3>
 
-    <input type="text" class="search" placeholder="{{t 'Filter users' }}" aria-label="{{t 'Filter users' }}"/>
+    <div class="input-append settings-table-search-section">
+        <input class="search input_field" type="text" placeholder="{{t 'Filter users' }}" aria-label="{{t 'Filter users' }}"/>
+        <button type="button" class="btn clear_search_button">
+            <i class="fa fa-remove" aria-hidden="true"></i>
+        </button>
```

Comment:
Thinking about this, I think we want to make a tiny template `clear_search_button.hbs` that we can use in all these places, similar to the existing `settings_checkbox.hbs`.  That would make it feel a lot more convenient to manage code around these.

Is there a way we can avoid needing to add classes to the parent element when adding these?  I'm just thinking about how to make it a maintainable component.

Reply:
I have created a template which include search box and clear search button. Please take a look.

> Is there a way we can avoid needing to add classes to the parent element when adding these? I'm just thinking about how to make it a maintainable component.

@timabbott Can you clear me this question, which classes are you talking about?